artiq/test/test_rpctool.py

```diff
@@ -0,0 +1,38 @@
+import os, sys
+import asyncio
+import unittest
+from artiq.protocols.pc_rpc import Server
+
+class Target:
+    def output_value(self):
+        return 4125380
+
+class rpctool_test(unittest.TestCase):
```

Comment:
Underscore separation is the naming convention for functions, not classes. How about ``TestRPCTool``?